docs/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clean": "gatsby clean",
     "develop": "gatsby develop",
-    "build": "cd .. && yarn && cd docs && gatsby build --prefix-paths"
+    "build": "cd .. && yarn && cd docs && NODE_ENV=production gatsby build --prefix-paths"
```

Comment:
This breaks on Windows. Not sure if that's a valid consideration, but IIRC Primer has been platform-agnostic until now.

Reply:
Good call; I'll update this to use `cross-env`.